fsharp-backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -864,10 +872,9 @@ and Fn =
     previewable : Previewable
     deprecated : Deprecation
     sqlSpec : SqlSpec
-    // TODO: reevaluate usefulness of below comment. at least rephrase.
-    // Functions can be run in JS if they have an implementation in this
-    // LibExecution. Functions whose implementation is in LibBackend can only be
-    // implemented on the server.
+
```

Comment:
This is duplicated.

Reply:
It was duplicated in the original code, so I kept it such. I can remove one, though - which?

Reply:
Either is fine.